pkg/kademlia/kademlia.go

```diff
@@ -109,11 +109,19 @@ func (k *Kademlia) Queried() {
 	k.lastQueried = time.Now()
 }
 
-// FindNear returns all nodes from a starting node up to a maximum limit
-// stored in the local routing table.
-func (k *Kademlia) FindNear(ctx context.Context, start storj.NodeID, limit int) (_ []*pb.Node, err error) {
+// FindNear returns a limited number of XOR closest nodes from the routing table
+// and the antechmaber respectively.
+func (k *Kademlia) FindNear(ctx context.Context, start storj.NodeID, routingTableLimit int, antechamberLimit int) (_ []*pb.Node, err error) {
```

Comment:
I think we actually want to return the RT nodes and antechamber nodes in separate structs